confidential-data-hub/secret/src/bin/secret_cli.rs

```diff
@@ -37,7 +38,7 @@ enum Cli {
 struct SealArgs {
     /// path of the file which contains the content to be sealed
     #[arg(short, long)]
-    file_path: String,
+    file_path: Option<String>,
```

Comment:
I think the reason to mark this field to `Option`, is that we want `Vault` secrets are set separately rather than by this cli tool. If so, we can move this field to `EnvelopeCommand`, thus that only Envelope type would use this field.

btw, in the original design, we want even for `VaultSecret`s this tool will help to set secret to remote service, like what `Envelope` does. But surely, this seems making this tool code logic complex. Thus I am ok with this PR, s.t. leave the setting to separate vault separately.

Reply:
Ah yeah I'll move that to the `EnvelopeCommand`.

I think we should enable provisioning secrets in the future, but that will take some work. I also think we might want to expose this functionality as part of a kubectl plugin.